orderer/common/server/main_test.go

```diff
@@ -55,11 +55,18 @@ type signerSerializer interface {
 	identity.SignerSerializer
 }
 
-// the path to cryptogen, which can be used by tests to create certificates
-var cryptogen, tempDir string
+// The path to cryptogen, configtxgen which can be used by tests to create certificates
+var cryptogen, configtxgen string
 
 func TestMain(m *testing.M) {
 	var err error
+
+	configtxgen, err = gexec.Build("github.com/hyperledger/fabric/cmd/configtxgen")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "configtxgen build failed: %v", err)
+		os.Exit(-1)
+	}
+
```

Comment:
We will not need this if use `func produceGenesisFileEtcdRaft(...)` instead of `generateBootstrapBlock`, which can be removed as well.